lib/xcodeproj/project/object/build_configuration.rb

```diff
@@ -168,9 +181,19 @@ def resolve_variable_substitution(key, value, root_target)
           when key
             # to prevent infinite recursion
             nil
+          when previous_key
+            # to prevent infinite recursion; we don't return nil as for the self recursion because it
```

Comment:
missing some words at the end?

Reply:
Dang, and I forgot exactly what I wanted to say. I pushed an update.